addOns/httpsInfo/httpsInfo.gradle.kts

```diff
@@ -11,9 +11,6 @@ zapAddOn {
 }
 
 dependencies {
-    implementation(files("lib/dvAPI_5.0.3.jar"))
-    implementation("org.slf4j:slf4j-log4j12:1.7.6") {
-        // Bundled in Deep Violet JAR.
-        exclude(group = "org.slf4j", module = "slf4j-api")
-    }
+    implementation("com.github.spoofzu:DeepViolet:5.1.16")
+    implementation("org.slf4j:slf4j-log4j12:1.7.6")
```

Comment:
We need to add this exclusion:
```
// Provided by ZAP.
exclude(group = "log4j")
```
(Missed that during the migration.)

Reply:
Done